x-pack/plugins/triggers_actions_ui/public/application/lib/alert_api.ts

```diff
@@ -93,18 +93,24 @@ export async function loadAlerts({
   });
 }
 
-export async function deleteAlert({ id, http }: { id: string; http: HttpSetup }): Promise<void> {
-  await http.delete(`${BASE_ALERT_API_PATH}/${id}`);
-}
-
 export async function deleteAlerts({
   ids,
   http,
 }: {
   ids: string[];
   http: HttpSetup;
-}): Promise<void> {
-  await Promise.all(ids.map(id => deleteAlert({ http, id })));
+}): Promise<{ successes: string[]; errors: string[] }> {
+  const successes: string[] = [];
+  const errors: string[] = [];
+  await Promise.all(ids.map(id => http.delete(`${BASE_ALERT_API_PATH}/${id}`))).then(
```

Comment:
I think if one delete fails, they will all report as failing.  Seems like this needs to be restructured so that the delete's are wrapped in a function that never fails, but internally, it updates `successes` and `errors`, so that it can report a combination of successes and failures.

Seems survivable for 7.7, and should probably just be a new tech debt issue.